bin/githubchart

```diff
@@ -56,7 +60,7 @@ end.parse!
 
 SVG_Path = ARGV.shift
 
-Chart = GithubChart.new(options).svg
+Chart = GithubChart.new(options).send(draw_method)
```

Comment:
Ah, I re-read what I did here, this makes sense now. I do think it might make sense to add in a method to githubchart.rb for `.render(type)` that checks supported_versions or some such, and then does the send